### DISCUSSION TITLE
When opening log view: Start with cursor at top og window?

### DISCUSSION THREAD

**Discussion by joakim-hove** [NeogitOrg/neogit#1467] [Q&A] [answered]
When I open the log view with "ll" the log is displayed nicely, however the initial position of the cursor is at the bottom of the window, is it possible to get it to automatically jump to the top of the window?

**Comment by CKolkey** [comment, marked as answer]
This is just a bug I haven't had time to fix, not the intended behaviour 😅

**Reply by joakim-hove** [reply]
Cool thank you - if you want to point me in the roughly right direction I am happy to give it a shot with a PR?

**Reply by CKolkey** [reply]
Sure! 
- The constructor for the buffer: https://github.com/NeogitOrg/neogit/blob/master/lua/neogit/buffers/log_view/init.lua#L66
- The buffer lib: https://github.com/NeogitOrg/neogit/blob/master/lua/neogit/lib/buffer.lua#L560

My guess is that it might have to do with the `:set_header` method in the buffer lib (that `<pageup>` call maybe), but don't take my word for it. Lemme know if you need anything else.
